conftest.py

```diff
@@ -63,6 +63,18 @@ def dtype(dtype_name) -> torch.dtype:
     return TEST_DTYPES[dtype_name]
 
 
+@pytest.fixture(scope='session')
+def torch_optimizer():
+    try:
+        import torch._dynamo as dynamo
+
+        torch.set_float32_matmul_precision('high')
+        return dynamo.optimize('inductor')
+    except Exception as er:
+        del er
```

Comment:
why delete  here?

Reply:
to not have a bare exception and not have an unused var too -- flake8 complains